src/lib/chat/index.ts

```diff
@@ -38,6 +38,7 @@ export interface IChatClient {
   searchMyNetworksByName: (filter: string) => Promise<MemberNetworks[] | any>;
   searchMentionableUsersForChannel: (channelId: string, search: string, channelMembers?: UserModel[]) => Promise<any[]>;
   getMessagesByChannelId: (channelId: string, lastCreatedAt?: number) => Promise<MessagesResponse>;
+  getMessageByChannelId: (channelId: string, messageId: string) => Promise<any>;
```

Comment:
Should we name this `getMessageByRoomId` since it's general for both Channels and Conversations?